docs/examples/cli/README.md

```diff
@@ -0,0 +1,22 @@
+### How to run bysquare in the terminal
+
+Before running bysquare, make sure you have Node.js installed. If you're using `nvm` (Node Version Manager) to manage your Node.js versions, you can set the minimum required Node version for this project with the following command:
+
+```bash
+nvm use
+```
+
+This step is optional, but it ensures that you're using the correct version of Node.js as specified in the .nvmrc file.
+
+To encode data from example.json and print it as a string in the terminal, use the following command:
+
+```bash
+npx bysquare --encode example.json
+```
+
+Make sure that path to example.json is correct and contains the data you want to encode.
+
+If you want to display the encoded data as a QR code in the terminal, you can pipe the output of the encode command to qrcode-terminal:
+```bash
+npx bysquare --encode example.json | npx qrcode-terminal
+```
```

Comment:
Toto sa mi páči, ale dáva to viac zmyslu pri vývoji. Väčšina používateľov knižníc netuší, čo je `nvm` a ako funguje, zároveň nemajú stiahnutý repozitár, aby mohli použiť `nvm use`. Bežne sa používa `npm install -g` alebo `npx`, a v týchto príkladoch by som to neriešil. 

Toto je skôr pre pokročilých používateľov, ktorí tieto veci ovládajú a rovno sa pozrú do `.sh` súboru. Základné informácie k CLI sú v hlavnom readme.

Reply:
[73a6548](https://github.com/xseman/bysquare/pull/35/commits/73a654874f3c9a29fe993879cfac45bdf2c9ef88)

Reply:
Nemusel si to hneď mazať tiež by ma zaujímal tvoj názor

Reply:
@xseman - prepáč za neskorú odpoveď. 

Súhlasím s tým, že niektorí používatelia nemusia vedieť, čo je `nvm`. Zároveň však predpokladám, že 99% užívateľov má stiahnutý repozitár a `nvm use` môžu použiť. Ako ináč by si stiahli example?

Osobne si myslím, že nie všetci užívatelia rozumejú `.sh` súborom a vedia v nich čítať. To sa týka podľa mňa len zdatnejších užívateľov z prostredia Linuxu a Macu.

Tiež treba brať do úvahy rozdiely v OS, ktoré užívatelia používajú. Napr.: Keď som predvčerom investigoval projekt, narazil som na `docs/uml/generate-images.sh` a nevedel som ho spustiť. Aby som ho spustil na Mac-u, musel som si nainštalovať `draw.io` cez `brew` a nasledne zmeniť v scripte `draw.io` na `/Applications/draw.io.app/Contents/MacOS/draw.io`. 

Best practice je z môjho pohľadu je skôr napísať nejaky markdown dokumnet s väčším počtom komentárov a copy paste code snippetov. To cieli na väčšie publikum.